pkg/api/controller.go

```diff
@@ -2859,12 +2861,9 @@ func (c *Controller) handleAPIErrorCallback(ctx context.Context, w http.Response
 	case errors.Is(err, authentication.ErrInsufficientPermissions):
 		c.Logger.WithContext(ctx).WithError(err).Info("User verification failed - insufficient permissions")
 		cb(w, r, http.StatusUnauthorized, http.StatusText(http.StatusUnauthorized))
-	case err != nil:
+	default:
```

Comment:
I don't understand this fix, and I believe it may be unrelated to this PR.  Why do we want to print an error and fail if called with err == nil?

Reply:
a local linter said that the err==nil is up at the start of this func, so checking just err not nil was something I could drop.

Reply:
Ouch, that's horrible... but at the same time a great catch. So thanks!